graph/export.go

```diff
@@ -61,6 +62,20 @@ func (s *TagStore) ImageExport(names []string, outStream io.Writer) error {
 				// and will not need to be added to this map
 				if len(repoTag) > 0 {
 					addKey(repoName, repoTag, img.ID)
+				} else {
+					//This must be an ID. Here repoName is an ID.
+					nonTruncID, err := s.graph.GetTruncIndex().Get(repoName)
+					if err != nil {
+						return err
+					}
+					reverseLookupMap := s.ByID()
+					for id, repos := range reverseLookupMap {
+						if id == nonTruncID {
+							for _, repo := range repos {
+								addKey(repo[:strings.Index(repo, ":")], repo[strings.Index(repo, ":")+1:], id)
```

Comment:
can you extract `strings.Index...` and assign it to a variable? it's kinda hard to read and duplicated